dist/redhat/scylla-machine-image.spec

```diff
@@ -58,6 +55,16 @@ install -m755 common/.bash_profile $RPM_BUILD_ROOT/home/scyllaadm
 %postun
 %systemd_postun scylla-image-setup.service
 
+%posttrans
+if [ -L /home/scyllaadm/.bash_profile ] && [ ! -e /home/scyllaadm/.bash_profile ]; then
+    rm /home/scyllaadm/.bash_profile
+    cp /etc/skel/.bash_profile /home/scyllaadm/
+    chown scyllaadm:scyllaadm /home/scyllaadm/.bash_profile
+    echo "" >> /home/scyllaadm/.bash_profile
+    echo "" >> /home/scyllaadm/.bash_profile
```

Comment:
same line as above...

Reply:
and it does nothing

Reply:
@syuu1228 ping

Reply:
It is just extra new lines for .bash_profile readability, can drop since it does noting

Reply:
> It is just extra new lines for .bash_profile readability, can drop since it does noting

echo -e '\n' >> /home/scyllaadm/.bash_profile

Reply:
Changed to `echo -e '\n'`, suggested by @amoskong